src/utils/warnings.ts

```diff
@@ -45,26 +50,29 @@ export function deprecationWarning(
 ) {
   if (condition) return;
 
-  if (process.env.NODE_ENV === 'production') {
-    return warn(
-      `DEPRECATION ${name} "${property}" -> ${
-        typeof betterAlternative === 'function'
-          ? betterAlternative()
-          : betterAlternative
-      }`,
-    );
-  }
+  return warn(
+    `DEPRECATION ${name} "${property}" -> ${
+      typeof betterAlternative === 'function'
+        ? betterAlternative()
+        : betterAlternative
+    }`,
+  );
 
   // we can make deprecations even better if we add the md syntax in the console.
   // anyway, everything down below will be stripped in the production build
   console.group(`⚠️ ${PREFIX}: Deprecation in ${name}`);
   warn(
     `"${property}" is deprecated, consider better alternative: ${
       typeof betterAlternative === 'function'
-        ? betterAlternative()
+        ? (betterAlternative as () => string)()
```

Comment:
Why did you add direct casting to function?

Reply:
I mean, what was the reason for casting the type? It looks like Typescript can manage it